colorbleed/plugins/maya/publish/collect_look.py

```diff
@@ -118,6 +120,7 @@ def collect(self, instance):
         # Collect file nodes used by shading engines (if we have any)
         files = list()
         if looksets:
```

Comment:
[Line 112](https://github.com/Colorbleed/colorbleed-config/pull/15/files#diff-3813032214deb50f51eaf9210aebdd55R112) gets the looksets by long names, but I believe `objectSets` are always their full path, because they cannot be i n a dag hierarchy. ;) So using `cmds.ls(x, long=True)` over the sets should be redundant...

Looking over the code here the order also looks slightly inlogical. Should we shift `looksets = cmds.ls(sets.keys(), long=True)` down and the logging of the found sets more up; so they are closer to their logic.

```python
self.log.info("Found the following sets:\n{}".format(sets.keys()))
```
This could be right below the collection of the sets (above attribute collection).

And the:
```python
looksets = cmds.ls(sets.keys(), long=True)
# which could also be:
looksets = sets.keys()
```
Could be directly above the `if looksets:` line.